### ISSUE TITLE
Drive Mounts Conflicts with Network Drives

### ISSUE THREAD

**Issue by DavidRockin** [maharmstone/btrfs#518]
So I'm not entirely sure if this is an issue with this BTRFS driver, OR if this is just another annoying weird quirk with Windows. 😔

But I have a network SMB folder mounted as the `P:` drive, and my one external SSD with BTRFS was mounted as `S:` Sometime over the past few days I was doing some file organization & cleanup, and consolidating data across multiple drives on some new drives I purchased. 

But a strange issue occurred where the mounted external SSD disappeared and I couldn't see it. Although I could see it in the Disk Manager, although as normally with BTRFS volumes I couldn't do anything with it. I've been confused until I was using KDE's Dolphin and noticed it was recognizing my drive, but not the network drive.

![image](https://user-images.githubusercontent.com/1907079/187495105-dd369b8f-9609-40eb-9877-e15bcbf664da.png)

In explorer the drive is missing; although while creating this issue I just noticed windows is confused thinking my network drive is BTRFS:

![image](https://user-images.githubusercontent.com/1907079/187496826-fe7f7206-2ec5-4f88-b5eb-f7b605017847.png)

And out of curiosity I turned on my server just to see what would happen, and it still lists it as a network drive, but then also says it's BTRFS, and reports the disk capacity of the network share BUT it still lists all file contents on my BTRFS SSD. Although I've changed the drive letter and things appear to be normal again.

----

I was using `1.8.0` when this happened, and upgrading to the latest `1.8.1` the same odd behaviour occurs. I just think it might be a good idea to have this documented if someone else experiences something similar with network drives & the BTRFS mounts.

**Comment by maharmstone** [comment]
I suspect this is just Windows weirdness - I've seen it get confused about network shares even without btrfs installed.

**Comment by Warrentheo** [comment]
Agreed, the easiest solution is to push drive letters away from the one that windows will assign next on its own...  Because of this I manually fill up drive letters from the bottom up to prevent this kind of issue.
